### PR TITLE
Form validation with JAVASCRIPT

### PR DESCRIPTION
**Pull request by eerapheal**
My Miroverce week 4 project and Form validation with JAVASCRIPT.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -250,3 +252,91 @@ function dissappearParent(e) {
...
+// email.value = localStorage.getItem('customerEmail');
+// formText.value = localStorage.getItem('customerInfo');
+
+// autoFill();
+
+// formName.onchange = autoFill;
+// email.onchange = autoFill;
+// formText.onchange = autoFill;
+
+// Full form validation person project
```
Comment (tufoinnkuo10) [inline script.js]: - [ ] please remove the commented code for clean code practices

File: script.js [hunk 2]
```diff
@@ -250,3 +252,91 @@ function dissappearParent(e) {
...
+    return showError(input, invalidMsg);
+  }
+  return true;
+}
+
+const form = document.querySelector('#contact-from-validation');
+
+const NAME_REQUIRED = 'Please enter your name';
+const EMAIL_REQUIRED = 'Please enter your email';
+const EMAIL_INVALID = 'Your email address should be in lowercase';
```
Comment (tufoinnkuo10) [inline script.js:303]: - [optional] kindly make the error message color to be red so it can easily be noticed

File: script.js [hunk 3]
```diff
@@ -11,7 +13,7 @@ document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click'
   navItem.classList.remove('active');
 }));
 
-// pop-up window
+// pop-up window in my see project section done with two javascript method.
 
 function displayProjects() {
```
Comment (tufoinnkuo10) [inline script.js:18]: - [optional] when your portfolio is launched your project popup comes up without me clicking on it. please fix that

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @eerapheal 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ email check ok

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by topeogunleye — APPROVED** [review]
### Hi @eerapheal, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
## [Optional] suggestions:
* Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**